autopush/router/gcm.py

```diff
@@ -22,39 +21,45 @@ def __init__(self, ap_settings, router_conf):
         self.min_ttl = router_conf.get("ttl", 60)
         self.dryRun = router_conf.get("dryrun", False)
         self.collapseKey = router_conf.get("collapseKey", "simplepush")
-        self.senderIDs = router_conf.get("senderIDs")
+        self.gcm = {}
+        self.senderIDs = {}
```

Comment:
You could actually combine the gcm and senderIDs dicts into one at this point, {sid: (auth, GCM()),} .. but wait a second..